DemoApp/android/build.gradle

```diff
@@ -3,16 +3,19 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Looks like blank app created via `react-native init` still uses v2.2.3. Is there a good reason that we'd better use v3.1.3? The inconvenience of bumping gradle version is Android Studio prompting users to download the newer version, so I am slightly prefer to stay on the default version, but I am open to other opinions.

Reply:
This was bumped for compatibility with firebase core and messaging dependencies. And since it is DemoApp which is used mostly for testing purpose, I think we can leave it that way.

Reply:
make sense, I am fine with using v3.